Polygon_IO/include/CGAL/IO/traits_multipolygon.h

```diff
@@ -0,0 +1,19 @@
+#ifndef CGAL_TRAITS_MULTIPOLYGON_H
+#define CGAL_TRAITS_MULTIPOLYGON_H
+#include <CGAL/Geometry_container.h>
+#include <boost/geometry/geometry.hpp>
+
+
+namespace boost{
+namespace geometry{
+namespace traits{
+// WKT traits for MultiPolygon
+template< typename R >
+struct tag<CGAL::Geometry_container<R, multi_polygon_tag> >
+{ typedef multi_polygon_tag type; };
+
+}//end traits
+}//end geometry
+}//end boost
+
+#endif 
```

Comment:
Please end the last line with a EOL, otherwise some tools may not treat the file as a valid text file.